logger.go

```diff
@@ -38,21 +30,70 @@ func (lvl LogLevel) String() (name string) {
 		name = "WARN"
 	case LogLevelError:
 		name = "ERROR"
+	default:
+		name = "invalid"
 	}
 	return
 }
 
-// logger is the base logging struct that handles writing to the logOutputter.
+func Debug(v ...interface{})                 { std.log(LogLevelDebug, v...) }
```

Comment:
Oh sorry when I switched from github.split view to normal diff, it looks much better...